src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespacebeforecasedefaultcolon/InputNoWhitespaceBeforeCaseDefaultColon.java

```diff
@@ -0,0 +1,106 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nowhitespacebeforecasedefaultcolon;
+
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+
+/*
+* Config = default
+*/
+public class InputNoWhitespaceBeforeCaseDefaultColon {
+    {
+        switch(1) {
+            case 1 : // violation
+                break;
+            case 2:
+                break;
+            default : // violation
+                break;
+        }
+
+        switch(2) {
+            case 2:
+                break;
+            default:
+                break;
+
+        }
+
+        switch(3) {
+            case 3/* space after */ : // violation
```

Comment:
I am not seeing any good/bad cases with a comment inbetween the `case` and the case's constant. Please add some to show they are handled fine. Comments have a tendency to be in very weird places in the AST tree.

Reply:
Done